web/services/nft_service.py

```
@@ -0,0 +1,249 @@
+
+# Use WEB3_URL_PROVIDER for Polygon RPC URL
+WEB3_URL_PROVIDER = settings.WEB3_PROVIDER_URL
+if not WEB3_URL_PROVIDER:
+    raise ValueError("❌ WEB3_PROVIDER_URL is missing in environment variables")
+
+BLOCKFROST_API_KEY = settings.BLOCKFROST_API_KEY
+BLOCKFROST_BASE_ENDPOINT = settings.BLOCKFROST_BASE_ENDPOINT
+
+
+class NFTBadgeService:
+    def __init__(self, contract_address, admin_private_key):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)